app/jobs/hyku_addons/reindex_available_works_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module HykuAddons
+  class ReindexAvailableWorksJob < ApplicationJob
+    def perform(array_of_cname)
```

Comment:
We can just call this `cnames`, so its a litte less cumbersome, and as its a single line block, could be moved up. 

```
def perform(cnames)
  cnames.each { |cname| fetch_work_save_cname(cname) }
end
```

Reply:
I want someone reading to know by just looking at it. Naming in a way that the intent or what is expected is obvious seems easier for clarity even if the other is shorted.

Reply:
I would provide an example of how to use the task in the rake file, so that the syntax and format do not have to be worked out by the reader, like so many of the hyku/hyrax/Addons tasks.

Reply:
We already have that comment in the rake file pushed with this PR.  However, if you open just the job file, you should  know what is expected without going to the rake task, since it is not unfathomable to think of scenarios that someone wants to use the job without the rake.

We can achieve that clarity by either adding a comment or naming things properly. 

I will change the name though and add comments to the job since the rake file already has that comment.